src/libslic3r/PrintObject.cpp

```diff
@@ -296,6 +302,7 @@ void PrintObject::prepare_infill()
     // Decide what surfaces are to be filled.
     // Here the stTop / stBottomBridge / stBottom infill is turned to just stInternal if zero top / bottom infill layers are configured.
     // Also tiny stInternal surfaces are turned to stInternalSolid.
+
```

Comment:
Removed.  
The change was delayed because I was looking into failures of fff_print_tests built with debug (but not release). Turns out that master branch behaves exactly the same, so I stopped checking.